offchain-modules/packages/app-cli/src/bridgeFee.ts

```diff
@@ -0,0 +1,192 @@
+  if (xchain !== 'ethereum') throw new Error('only support ethereum currently');
+  const configPath = nonNullable(opts.config);
+
+  nconf.env().file({ file: configPath });
+  const cfg: Config = nconf.get('forceBridge');
+  await new ForceBridgeCore().init(cfg);
+  const conn = await getDBConnection();
+  const bridgeFeeDB = new BridgeFeeDB(conn);
+  const ethAsset = new EthAsset(asset);
+
+  const withdrawedBridgeFee = await bridgeFeeDB.getEthTotalWithdrawedBridgeFee(asset);
+  const humanizedWithdrawedBridgeFee = ethAsset.getHumanizedDescription(withdrawedBridgeFee);
+  console.log('total withdrawed bridge fee:', humanizedWithdrawedBridgeFee);
+}
+
+async function getTotalAvailable(command: commander.Command) {
+  const opts = command.opts();
+  const xchain = nonNullable(opts.xchain);
+  const asset = nonNullable(opts.asset);
+  if (xchain !== 'ethereum') throw new Error('only support ethereum currently');
+  const configPath = nonNullable(opts.config);
+
+  nconf.env().file({ file: configPath });
+  const cfg: Config = nconf.get('forceBridge');
+  await new ForceBridgeCore().init(cfg);
+  const conn = await getDBConnection();
+  const bridgeFeeDB = new BridgeFeeDB(conn);
+  const ethAsset = new EthAsset(asset);
+
+  const withdrawedBridgeFee = await bridgeFeeDB.getEthTotalWithdrawedBridgeFee(asset);
+  const generatedBridgeFee = await bridgeFeeDB.getEthTotalGeneratedBridgeFee(asset);
+  const availableBridgeFee = new Amount(generatedBridgeFee, 0).sub(new Amount(withdrawedBridgeFee, 0)).toString(0);
+  const humanizedAvailableBridgeFee = ethAsset.getHumanizedDescription(availableBridgeFee);
+  console.log('total available bridge fee:', humanizedAvailableBridgeFee);
+}
+
+async function generateWithdrawTxSignature(command: commander.Command) {
+  const opts = command.opts();
+  const xchain = nonNullable(opts.xchain);
+  if (xchain !== 'ethereum') throw new Error('only support ethereum currently');
+  const asset = nonNullable(opts.asset);
+  const ethAsset = new EthAsset(asset);
+  const recipient = nonNullable(opts.recipient);
+  const amount = nonNullable(opts.amount);
+  if (recipient.length !== amount.length) throw new Error('recipient number should equal amount number');
+  const configPath = nonNullable(opts.config);
+  nconf.env().file({ file: configPath });
+  const cfg: Config = nconf.get('forceBridge');
+  await new ForceBridgeCore().init(cfg);
+
+  const withdrawRecords = recipient.map((r, i) => {
+    return {
+      ckbTxHash: WithdrawBridgeFeeTopic,
+      token: asset,
+      amount: ethAsset.parseAmount(amount[i]),
+      recipient: r,
+    };
+  });
+  const ethChain = new EthChain('verifier');
+  const message = await ethChain.getUnlockMessageToSign(withdrawRecords);
+  const privKeyPath = ForceBridgeCore.config.eth.multiSignKeys[0].privKey;
+  const privKey = parsePrivateKey(privKeyPath);
+  const { v, r, s } = ecsign(Buffer.from(message.slice(2), 'hex'), Buffer.from(privKey.slice(2), 'hex'));
```

Comment:
Don't need multi-sig?

Reply:
Every verifier use `forcecli fee generate-withdraw-tx-signature` cli to generate signature, then collector can ask verifiers for signatures and send withdraw-bridge-fee-tx with these signatures (`forcecli fee send-withdraw-tx -s [signatures] ...`).